pipelines/lib/util.groovy

```diff
@@ -1994,6 +1994,73 @@ def void runDispatchVerify(Map p) {
   } // withEnv
 } // runDispatchVerify
 
+/**
+ * Convert Gen 3 results into a form suitable for dispatch-verify.
+ *
+ * Example:
+ *
+ *     util.runGen3ToJob(
+ *       gen3Dir: gen3Dir,
+ *       collectionName: collectionName,
+ *       namespace: "",
+ *       datasetName: datasetName,
+ *     )
+ *
+ * @param p Map
+ * @param p.gen3Dir String Path to the Gen 3 repository
+ * @param p.collectionName String The collection to search for metrics.
+ * @param p.namespace String The metrics namespace to filter by, e.g. validate_drp, or "" for all metrics.
+ * @param p.datasetName String The dataset name. Eg., validation_data_cfht
+ */
+def void runGen3ToJob(Map p) {
+  util.requireMapKeys(p, [
+    'gen3Dir',
+    'collectionName',
+    'namespace',
+    'datasetName',
+  ])
+
+  def run = {
+    util.bash '''
+      set +o xtrace
+      source /opt/lsst/software/stack/loadLSST.bash
+      setup verify
+      set -o xtrace
+
+      if [[ -n $METRIC_NAMESPACE ]]
```

Comment:
I would usually write this as `if [[ ... ]]; then`, keeping the control distinct from the actions, but this syntax is OK.